### DISCUSSION TITLE
Eyes/brightness solutions/troubleshooting external screen

### DISCUSSION THREAD

**Discussion by DashingDave** [MonitorControl/MonitorControl#1543] [Q&A]
Without going into my eye issues and my current awkward workaround on my iMac (2017 27" Retina 5K, in case that matters), here's what I'm trying to accomplish with an external monitor:
- One screen/monitor for applications that have a native Dark Mode
- One screen/monitor for applications that don't (and for files that don't have dark backgrounds, which is basically anything that would be read by other people: PDFs, word processing docs, artwork, etc.)

From what I can find on the internet, I can't set just one of the two screens to Invert Colors mode, so I'm trying to use a 2nd-monitor-adjustment app (BetterDisplay, Black Light, Lunar, and MonitorControl are the ones I'm trying to figure out) to see what I could do to a second screen that might make the "Light" stuff easier for my eyes to handle.

Unfortunately, I can't get any of those apps to work with my available setup: Airplay to an AppleTV with 10+ years old Sony TV and Airplay to a 2022ish Roku TV. Making any adjustments to the settings has had no effect on the second screen.
- Is Airplay the issue–is a physical connection required? Or maybe its older or Android-OS TVs?

If I can get this type of app working, does anyone have any suggestions on what settings I could use on the second screen that would make it easier to deal with the apps and windows that are basically giant rectangle of white? Is there a way to do a black-for-white swap without using Invert Colors?

I don't really know how to make use of these applications' capabilities, so any advice or links to information that might help me would be appreciated.

Thanks!

**Comment by waydabber** [comment]
You can invert the screen on a per display level with BetterDisplay. Use Image Adjustments / Invert Color for no-native invert. Hold OPTION while opening Image Adjustments to access the per-display framebuffer level invert feature.

<img width="335" alt="Screenshot 2024-05-02 at 18 07 35" src="https://github.com/waydabber/BetterDisplay/assets/37590873/27fc6eea-8c6e-4278-bfe6-6e991bfc38d3">
